java/ql/src/experimental/Security/CWE/CWE-665/InsecureRmiJmxEnvironmentConfiguration.qhelp

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>An improperly set environment variable during the creation of an RMI or JMX server can lead 
+to an unauthenticated remote code execution vulnerability. This is because the
+RMI/JMX server environment allows attackers to supply arbitrary objects to the authentication 
+method, resulting in the attempted deserialization of an attacker-controlled object.
```

Comment:
```suggestion
method, resulting in the attempted deserialization of an attacker-controlled object.
</p>
```

Reply:
And various other places: make sure paragraphs are surrounded by `<p></p>` and all elements are terminated.

Reply:
The XML is validated, we also addressed [these concerns](https://github.com/github/codeql/pull/5811#discussion_r631729679) and overall tried to improve the help